cloud-functions/upload_fhir_bundle/main.py

```diff
@@ -87,9 +83,8 @@ def upload_fhir_bundle(request: flask.Request) -> flask.Response:
 
     # Upload bundle to the FHIR store using the GCP Crednetial Manager for
     # authentication.
-
     response = upload_bundle_to_fhir_server(
         request_body.bundle, credential_manager, fhir_store_url
     )
```

Comment:
Only realizing as I do this review that `response` is a `dict` not a `flask.Response` object. This is a mistake I made when originally writing this  function. I think it was before we moved to returning `flask.Responses`

Reply:
I think this is ok here though.  It seems to still function like a flask.response object.  And we are only interested if the upload was successful or not.  I can do some reconstructing here if we need to, but maybe in a different PR/Branch though.  ;)